src/commands/deleteSuggestion.ts

```diff
@@ -34,6 +40,18 @@ export const command: IBotCommand = {
 
             return interaction.editReply({ embeds: [errorMessageEmbed] });
         }
+        
+        const member=suggestionMessage.author
+
+        const dmEmbed = new MessageEmbed()
+                .setColor("YELLOW")
+                .setTitle("Your suggestion has been deleted")
+                .setDescription(`
+                    Reason: ${interaction.options.getString("reason", true)}
+                    By: ${interaction.member.user.username}
+                `)
+
+        await member?.send({ embeds: [dmEmbed] });
```

Comment:
`suggestionMessage.author` returns the bot's user

Reply:
Yeah I changed the suggestionMessage's author to the one who made the suggestion not the bot's but that will cause some messageEmbed errors

Reply:
I tried to get author's tag from embed title then grab it's user from users cache
```ts
const suggestionTitleSplit = suggestionMessage?.embeds[0].title?.split(' - ');
const suggestionAuthor = suggestionTitleSplit ? client.users.cache.find(u => 
    u.tag === suggestionTitleSplit[1]
) : undefined;
```